docs/UserGuide.md

```diff
@@ -229,6 +229,8 @@ Application: Used to add new tasks for tracking purposes.
 
 Format: `add n/TASK NAME m/MODULE d/DESCRIPTION [a/START TIME] b/DEADLINE w/WORKLOAD [r/RECURRENCE] [t/TAG]…​`
 
+* If you wish to include any of the optional field, you need to follow its format.
```

Comment:
Not sure what you mean by "follow its format"? Does it mean that the optional parameters have to be in the order shown in the format?

Reply:
I move this line to note of command and delete it in addCommand. It only means, you cannot give an invalid format for the optional field, just because it is optional. Otherwise, some may think since it is optional, the MB will ignore the wrong format and proceed with the rest of the command.